Examples/Io/Csv/src/CsvOutputData.hpp

```diff
@@ -114,21 +114,39 @@ struct HitData {
                  x, y, z, t);
 };
 
+struct MeasurementData {
+  /// Event-unique measurement identifier. Each value can appear at most once.
+  uint64_t hit_id;
+  /// Hit surface identifier.
+  uint64_t geometry_id = 0u;
+  /// Partially decoded hit surface identifier components.
+  uint32_t volume_id, layer_id, module_id;
```

Comment:
Do we really want to store this? It was in the TrackML dataset only to help the participants. It is duplicated information for the purpose of data storage.

Reply:
Happy to go back to geometry_id only.